### PR TITLE
test: add ability to run EF spec tests

### PR DESCRIPTION
**Pull request by MegaRedHand**
Closes: #16

### CODE REVIEW

File: test/test_helper.exs [hunk 1]
```diff
@@ -1 +1,60 @@
...
+          config == pinned_config do
+        test_name = "c:#{config} f:#{fork} r:#{runner} h:#{handler} s:#{suite} -> #{cse}"
+
+        test_runner = Map.get(SpecTestUtils.runner_map(), runner)
+
+        @tag :spectest
+        @tag config: config
+        @tag fork: fork
+        @tag runner: runner
+        @tag suite: suite
```
Comment (mpaulucci) [inline test/test_helper.exs:46]: very cool!

File: test/test_helper.exs [hunk 2]
```diff
@@ -1 +1,60 @@
...
+        if test_runner == nil do
+          @tag :skip
```
Comment (mpaulucci) [inline test/test_helper.exs:48]: This is really cool, but I think we need a more fine grained option to skip/include tests, so we can start adding them one by one, or runner by runner.

**Review by mpaulucci — APPROVED** [review]
👏

File: test/spec-test/runners/ssz_test.exs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+defmodule SSZTestRunner do
```
[the remark this answers is not in the document]
Comment (MegaRedHand) [reply test/spec-test/runners/ssz_test.exs]: I like the idea!

File: test/test_helper.exs (hunk 2 above)
Comment (mpaulucci) [reply test/test_helper.exs:48]: sounds good! Would it work with the @skip tag like the tests that don't have a runner?

Comment (MegaRedHand) [reply test/test_helper.exs:90]: Yeah!

Comment (mpaulucci) [reply test/test_helper.exs:90]: nice!

**Pull request merged**